components/Card.js

```diff
@@ -95,11 +106,37 @@ export default function Card({ date, id, weights, handleDelete, heights, feastTi
           </AttributeList>
         </>
       )}
+      {diaperColors.length !== 0 && (
+        <>
+          <AttributeText>Diaper Color</AttributeText>
+          <AttributeColorList>
+            {diaperColors.map((diaperColor) => (
+              <AttributeColorListItem key={diaperColor._id}>
+                {pathname === '/' && (
+                  <DeleteSingleColor
+                    inputColor={diaperColor.value}
+                    aria-label="delete single data point"
+                    onClick={() => handleDelete(id, diaperColor._id, 'diaperColors')}
+                  >
+                    <X fontSize="1.5em" alt="delete" />
+                  </DeleteSingleColor>
+                )}
+                <ColorRoundCase inputColor={diaperColor.value}>
+                  {/* ToDo swap hex number for color name */}
+                  <SrOnly>{diaperColor.value}</SrOnly>
```

Comment:
What is this component for? Is this for screen readers? I'm not sure if it really needs a component for that usecase. Maybe you can just use
`ariaLabel={diaperColor.value}` or `aria-label={diaperColor.value}`
on the ColorRoundCase component?

Reply:
this is a really good idea!